jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorTest.java

```diff
@@ -187,6 +187,36 @@ void customize_inKubernetes_shouldReturnNativeUbiFrom() throws IOException {
       assertBuildFrom(resultImages, "registry.access.redhat.com/ubi8/ubi-minimal:8.6");
     }
 
+    @Test
+    @DisplayName("in Kubernetes with native artifact, should disable Jolokia")
+    void customize_inKubernetes_shouldDisableJolokia() throws IOException {
+      // Given
+      in(RuntimeMode.KUBERNETES);
+      withNativeBinaryInTarget();
+      // When
+      final List<ImageConfiguration> resultImages = new QuarkusGenerator(ctx).customize(new ArrayList<>(), true);
+      // Then
+      assertThat(resultImages).singleElement()
+        .extracting("buildConfiguration.env")
+        .asInstanceOf(InstanceOfAssertFactories.map(String.class, String.class))
+        .containsEntry("AB_JOLOKIA_OFF", "true");
+    }
+
+    @Test
+    @DisplayName("in Kubernetes with native artifact, should disable Prometheus")
+    void customize_inKubernetes_shouldDisablePrometheus() throws IOException {
+      // Given
+      in(RuntimeMode.KUBERNETES);
+      withNativeBinaryInTarget();
+      // When
+      final List<ImageConfiguration> resultImages = new QuarkusGenerator(ctx).customize(new ArrayList<>(), true);
+      // Then
+      assertThat(resultImages).singleElement()
+        .extracting("buildConfiguration.env")
+        .asInstanceOf(InstanceOfAssertFactories.map(String.class, String.class))
+        .containsEntry("AB_JOLOKIA_OFF", "true");
```

Comment:
Shouldn't it be this instead?
```suggestion
        .containsEntry("AB_PROMETHEUS_OFF", "true");
```

Reply:
:facepalm: